api/v1/db/oracledb/games-dao.js

```diff
@@ -113,22 +113,22 @@ const postGame = async (body) => {
 };
 
 /**
- *
- * @param {id} developerId id of developer record to check existance
+ * @summary Checks if a developer record with an id that matches the passed in developerId exists
+ * @param {string} developerId id of developer record to check existance
  * @returns true if developer record is found and false if no records are found
  */
 const isValidDeveloper = async (developerId) => {
   const sqlParams = { id: developerId };
   const sqlQuery = `
-    SELECT ID AS "id"
+    SELECT COUNT(ID)
     FROM DEVELOPERS
     WHERE ID = :id
   `;
 
   const connection = await conn.getConnection();
   try {
     const response = await connection.execute(sqlQuery, sqlParams);
-    return !_.isEmpty(response.rows);
+    return response.rows[0]['COUNT(ID)'] > 0;
```

Comment:
You can still use the alias to make sure all of your SQL queries follow the same coding style.